README.md

```diff
@@ -258,6 +276,129 @@ Supported commands are: `act()`, `click()`, `get()`, `locate()`, `mouse_move()`
 
 </details>
 
+
+### 🔧 Custom Models
+
+You can create and use your own models by subclassing the `ActModel` (used for `act()`), `GetModel` (used for `get()`), or `LocateModel` (used for `click()`, `locate()`, `mouse_move()`) classes and registering them with the `VisionAgent`.
+
+Here's how to create and use custom models:
+
+```python
+from askui import (
+    VisionAgent,
+    ActModel,
+    GetModel,
+    LocateModel,
+    ModelRegistry,
+    ResponseSchema,
+    ImageSource,
+    Locator,
+    Point,
+)
+from typing import Type
+from PIL import Image
+
+# Define custom models
+class MyActModel(ActModel):
+    def act(self, goal: str, model_choice: str) -> None:
+        # Implement custom act logic, e.g.:
+        # - Use a different AI model
+        # - Implement custom business logic
+        # - Call external services
+        print(f"Custom act model executing goal: {goal}")
+
+# Because Python supports multiple inheritance, we can subclass both `GetModel` and `LocateModel` (and even `ActModel`)
+# to create a model that can both get and locate elements.
+class MyLocateModel(GetModel, LocateModel):
+    def get(
+        self,
+        query: str,
+        image: ImageSource,
+        response_schema: Type[ResponseSchema] | None,
+        model_choice: str,
+    ) -> ResponseSchema | str:
+        # Implement custom get logic, e.g.:
+        # - Use a different OCR service
+        # - Implement custom text extraction
+        # - Call external vision APIs
+        return f"Custom response to query: {query}"
+
+
+    def locate(
+        self,
+        locator: str | Locator,
+        image: ImageSource,
+        model_choice: ModelComposition | str,
+    ) -> Point:
+        # Implement custom locate logic, e.g.:
+        # - Use a different object detection model
+        # - Implement custom element finding
+        # - Call external vision services
+        return (100, 100)  # Example coordinates
+
+# Create model registry
+custom_models: ModelRegistry = {
+    "my-act-model": MyActModel(),
+    "my-get-model": MyGetModel(),
+    "my-locate-model": MyLocateModel(),
+}
```

Comment:
So I can have my standard models and my custom models or?

Reply:
Yes. See examples below in the README.